sapphire/api.py

```diff
@@ -438,6 +440,57 @@ def validate_numbers(country=None, cluster=None, subcluster=None):
             raise Exception('Invalid subcluster number, '
                             'must be multiple of 100.')
 
+    def uptime(self, stations=None, start=None, end=None):
+        """
+        adapted from topaz 150805_n_active
+        returns the number of HOURS all stations have simultaneous data/events
+
+        :param stations: station id or a list of station ids
+        :param start, end: start, end timestamp
+        :returns: number of hours with simultaneous data
+        """
+        data = {}
+
+        if stations is None:
+            assert False
```

Comment:
I was considering to allow ```Network().uptime()``` to get the full network uptime. But this is nonsense (Hence the return 0 instead of calculating it).
This is a left-over from implementing in the ```BaseCluster``` class, in that context ```ScienceParkCluster().uptime()``` would make sense. 

I'll remove it after debugging.